clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -23,22 +28,48 @@ export const CustomTextInput = ({
   ...props
 }: InputProps & FieldHookConfig<string>) => {
   const [field, meta] = useField(props);
-  const { type, placeholder } = props;
+  const { type: initialType, placeholder } = props;
```

Comment:
I debated whether this should be a new component or a "mode" of the text input. I don't expect this component to gain any other modes like this, so I think this the convenient option.